buildSrc/src/main/java/com/uber/okbuck/core/dependency/ExternalDependency.java

```diff
@@ -57,7 +58,7 @@ public String getCacheName() {
     public String getCacheName(boolean useFullDepName) {
         if (useFullDepName) {
             if (!StringUtils.isEmpty(group)) {
-                return group + "." + depFile.getName();
+                return group + "--" + name + "--" + String.valueOf(version) + "." + packaging;
```

Comment:
lets move `--` to a constant so we can change it easier if needed

Reply:
done